src/constants/index.tsx

```diff
@@ -446,6 +480,15 @@ export const NETWORK_DETAIL: { [chainId: number]: NetworkDetails } = {
   },
 }
 
+export const RPC_URLS: { [chainId: number]: string } = {
+  [ChainId.MAINNET]: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`,
+  [ChainId.XDAI]: 'https://rpc.gnosischain.com/',
```

Comment:
XDAI and RINKEBY are deprecated so we should use GNOSIS / GOERLI

Reply:
Good point and that needs to be changed for sure, although I think it's out of scope of this PR and another new issue can be created specifically for this.